src/main/java/teammates/common/util/Const.java

```diff
@@ -615,10 +615,16 @@ private Const() {
         public static final String INSTRUCTOR_PERMISSION_ROLE_CUSTOM = "Custom";
     }
 
-    public static class GenderTypes {
-        public static final String MALE = "male";
-        public static final String FEMALE = "female";
-        public static final String OTHER = "other";
+    public enum Gender {
+        MALE, FEMALE, OTHER;
+
+        public static Gender gendercorrection(String gender) {
```

Comment:
`gendercorrection` is a rather weird name. If the intention is to parse gender from String input, then it should be named exactly that: `parse`.